pkpdapp/pkpdapp/models/prior.py

```diff
@@ -0,0 +1,55 @@
+#
+# This file is part of PKPDApp (https://github.com/pkpdapp-team/pkpdapp) which
+# is released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+
+from pkpdapp.pkpdapp.models import variable
+from django.db import models
+from pkpdapp.models import Variable
+from django.db.models import Q
+
+
+class Prior(models.Model):
+    """
+    Variable prior in inference
+    """
+
+    variable = models.ForeignKey(
```

Comment:
I'm guessing that you mean that this prior is specifying the prior for one of the variables you are performing inference on? Then I think this would be a one-to-one relationship (https://docs.djangoproject.com/en/3.2/topics/db/examples/one_to_one/), unless you can have multiple priors on the same variable?

Reply:
I think the idea was to have a Prior model which determined which type of prior was being used for a given parameter. The problem then becomes how do you specify the parameters of that prior, since some priors take one parameter (e.g. an exponential) whereas others take more (e.g. normal). So the thinking behind a PriorParameter was to have potentially a number of such models bound to a singe Prior.

Reply:
yup, that makes sense. But what is the `variable` field in `Prior`?

Reply:
Variable was just to map it to a parameter which we were doing inference over?

Reply:
Sure, sounds good. Should this be a one-to-one relationship though, I don't think it makes sense for a variable to have multiple priors?

Reply:
Yep, it should.